windows/python-setup/python-exe.bat

```diff
@@ -1,13 +1,18 @@
 @echo off
+
+Set version=3.4.3

```

Comment:
Not necessary to fix now, but next time we do a version bump of the bundled Python installer, let's instead change the filename to just `python3-installer-32bit.msi` so we don't have to worry about this hard-coded version string.

Reply:
Ok, @benjaoming, I will find ways, next time on how to version bump of the bundled Python installer.